python/lsst/meas/algorithms/psfCandidate.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from ._psfCandidate import PsfCandidateF
+
+__all__ = []  # import only for side effects
+
+PsfCandidateF.getCandidateRating = PsfCandidateF._getCandidateRating
+
+def setCandidateRating(rating):
+    raise NotImplementedError("must not exist for this type \
+            since getCandidateRating is calculated")
+
+PsfCandidateF.setCandidateRating = PsfCandidateF._getCandidateRating
```

Comment:
Also, it's weird (but presumably out of scope) that the C++ is so different. In C++ `PsfCandidate.setCandidateRating` uses the implementation from `SpatalCellCandidate` instead of raising an exception. If the C++ did something sensible it might be possible to get rid of this .py file.